src/msw/window.cpp

```diff
@@ -3707,57 +3707,68 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
         case WM_NCCALCSIZE:
             {
                 const wxBorder border = TranslateBorder(GetBorder());
+
                 if (wxUxThemeIsActive() && border == wxBORDER_THEME)
                 {
+                    RECT newrcClient; // new client rect
+
                     // first ask the widget to calculate the border size
                     rc.result = MSWDefWindowProc(message, wParam, lParam);
                     processed = true;
 
-                    // now alter the client size making room for drawing a
-                    // themed border
-                    RECT *rect;
-                    NCCALCSIZE_PARAMS *csparam = NULL;
-                    if ( wParam )
-                    {
-                        csparam = (NCCALCSIZE_PARAMS *)lParam;
-                        rect = &csparam->rgrc[0];
-                    }
-                    else
+                    if (wParam)
                     {
-                        rect = (RECT *)lParam;
+                        LPNCCALCSIZE_PARAMS pnccsp = (LPNCCALCSIZE_PARAMS)lParam;
+                        // pnccsp->rgrc[0] is the computed new client rect
+                        // pnccsp->rgrc[1] is going to be the dst blit rect
+                        // pnccsp->rgrc[2] is going to be the src blit rect
+
+						//::CopyRect(&newrcClient, &pnccsp->rgrc[0]);
```

Comment:
in the first place, I used `::CopyRect` instead of the direct `=` (Assignment Operator), because I was not sure which approach to use and because wxWidgets code itself sometimes uses the first one and sometimes uses the other one. and I this that I will leave `=` (Assignment Operator) and remove `::CopyRect`. What do you think?